src/js/components/TabForm.js

```diff
@@ -21,6 +24,7 @@ class TabForm extends React.Component {
     });
 
     this.triggerSubmit = function () {};
+    this.isValidated = false;
```

Comment:
Let's not set properties on the instance of the class, this is a bad practice. We can either opt to use internal storage or state if it makes sense to re-render.